File: Algoritmos, Estrutura de Dados e Linguagens/Exercises/0000000024.pas
PROGRAM ex0000000024;
USES CRT;
VAR reais,dol,marcoalem,libraester : real;
BEGIN
    readln(reais);
    dol := reais*1.80;
    marcoalem := reais*2.00;
    libraester := reais*3.57;
    writeln('Dolar:',dol:2:2);
    writeln('Marco Alemao:',marcoalem:2:2);
    writeln('Libra Esterlina:',libraester:2:2);
END.

File: Algoritmos, Estrutura de Dados e Linguagens/Algoritmos e Estrutura de Dados/Exercicios/C/0000000024.c
// Fonte: Fundamentos da Programação de Computadores: Algoritmos, Pascal, C, C++ e Java
// Exercício: Faça um programa que receba a quantidade de dinheiro em reais que uma pessoa que vai viajar possui. Ela vai passar por vários países e precisa converter seu dinheiro em dólares, marco alemão e libra esterlina. sabe-se que a cotação do dólar é de R$ 1,80; do marco alemão, de R$ 2,00; e da libra esterlina, de R$ 3,57. O programa deve fazer as conversões e mostrá-las.

#include <stdio.h>

int main(){
    float real;
    printf("Quantidade que tem para viajar: ");scanf("%f",&real);
    float dolar = 1.80 * real, marcoalemao = 2.0 * real, libraesterlina = 3.57 * real;
    printf("Dolar: %.2f\n",dolar);
    printf("Marco Alemao: %.2f\n",marcoalemao);
    printf("Libra Esterlina: %.2f\n",libraesterlina);
}